hellogsm-web/src/main/java/team/themoment/hellogsm/web/domain/application/service/impl/ApplicationListQueryImpl.java

```diff
@@ -13,12 +13,22 @@
 import team.themoment.hellogsm.web.domain.application.repository.ApplicationRepository;
 import team.themoment.hellogsm.web.domain.application.service.ApplicationListQuery;
 
+/**
+ * 모든 원서를 paging하여 조회하는 service implementation 입니다.
+ */
 @Service
 @XRayEnabled
 @RequiredArgsConstructor
 public class ApplicationListQueryImpl implements ApplicationListQuery {
     final ApplicationRepository applicationRepository;
 
+    /**
+     * 원서들을 pagination하여 반환해줍니다.
```

Comment:
```suggestion
     * 원서들을 pagination하여 반환해 줍니다.
```
여기도 마찬가지로 띄어쓰기 수정해주세요.

Reply:
수정하였습니다.